src/librustc_typeck/check/mod.rs

```diff
@@ -2423,7 +2423,7 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
 
         fn parameter_count_error<'tcx>(sess: &Session, sp: Span, expected_count: usize,
                                        arg_count: usize, error_code: &str, variadic: bool,
-                                       def_span: Option<Span>) {
+                                       def_span: Option<Span>, sugg: Option<String>) {
```

Comment:
Nit: can you add a comment to this method and explain what some of the arguments are? Or perhaps just give this a clearer name, like `suggest_unit_argument`? (In that case, why not make it a bool?)

Alternatively, we could make this more general, but right now the added labels only make sense with `()`.

Reply:
done